docs/src/examples.md

```diff
@@ -12,7 +12,7 @@ The high-level outline of performing a simulation is:
 1. Create a [`Crystal`](@ref), either by providing explicit geometry information
     (Example 1), or by loading a `.cif` file (Example 2).
 2. Using the `Crystal`, construct a collection of [Interactions](@ref).
```

Comment:
Perhaps this should be lower case too.  I think the reference isn't working.  (Clicking the linking gives me 404.)

Reply:
Odd the reference appears to be working for me -- this one needs be capital for the `@ref` to work since it's referring to a section title.